src/plugins/vis_type_timeseries/server/plugin.ts

```diff
@@ -115,6 +120,13 @@ export class VisTypeTimeseriesPlugin implements Plugin<VisTypeTimeseriesSetup> {
       searchStrategyRegistry,
     };
 
+    setDataSourceEnabled({ enabled: !!plugins.dataSource });
+    core.savedObjects.addClientWrapper(
+      11,
```

Comment:
Can we give the number a more meanful name?

Reply:
Refactored to `TIMESERIES_VISUALIZATION_CLIENT_WRAPPER_PRIORITY`